components/script/script_runtime.rs

```diff
@@ -761,22 +760,26 @@ unsafe extern "C" fn gc_slice_callback(
 ) {
     match progress {
         GCProgress::GC_CYCLE_BEGIN => GC_CYCLE_START.with(|start| {
-            start.set(Some(now()));
+            start.set(Some(SystemTime::now()));
             println!("GC cycle began");
         }),
         GCProgress::GC_SLICE_BEGIN => GC_SLICE_START.with(|start| {
-            start.set(Some(now()));
+            start.set(Some(SystemTime::now()));
             println!("GC slice began");
         }),
         GCProgress::GC_SLICE_END => GC_SLICE_START.with(|start| {
-            let dur = now() - start.get().unwrap();
+            let dur = SystemTime::now()
+                .duration_since(start.get().unwrap())
+                .unwrap();
```

Comment:
I think we should try to not panic here, so please provide defaults if we cannot unwrap successfully.

Reply:
Changed the last `unwrap` to `unwrap_or_default`, for lack of a smarter solution.